src/Controller/TaskController.java

```diff
@@ -480,7 +480,7 @@ protected void updateItem(final Task item, final boolean empty)
         TableColumn<Task, String> nameColumn = new TableColumn<>("Task");
         nameColumn.setCellValueFactory(new PropertyValueFactory<>("name"));
 
-        TableColumn<Task, String> assignmentColumn = new TableColumn<>("Assignments");
+        TableColumn<Task, String> assignmentColumn = new TableColumn<Task,String>("Assignments");
```

Comment:
This change does not seem correct.  The Java compiler will properly infer the type argument of the `TableColumn` constructor.  There is no need to add an explicit type specification here.